src/app/RecordingList/ActiveRecordingsList.tsx

```diff
@@ -66,6 +67,11 @@ export const ActiveRecordingsList: React.FunctionComponent<ActiveRecordingsListP
   const [errorMessage, setErrorMessage] = React.useState('');
   const { url } = useRouteMatch();
 
+  const RECORDING_CREATED = "RecordingCreated";
+  const RECORDING_DELETED = "RecordingDeleted";
+  const RECORDING_SAVED = "RecordingSaved";
+  const RECORDING_ARCHIVED = "RecordingArchived";
```

Comment:
Could these be made into an enum? TypeScript has syntax like this:

```
enum NotificationCategory {
  RecordingCreated = 'RecordingCreated',
  ...
}
```

which I think would work nicely here.

Reply:
Fixed with the most recent push, thanks